### DISCUSSION TITLE
Provide Unflatten Graphviz Command

### DISCUSSION THREAD

**Discussion by saulshanabrook** [hpcc-systems/hpcc-js-wasm#71] [Q&A]
Does this library support the [`unflatten`](https://www.graphviz.org/pdf/unflatten.1.pdf) graphviz command that is useful for graphs with multiple unconnected parts? For example, it is provided by the [a Python wrapper around Graphviz](https://graphviz.readthedocs.io/en/stable/api.html#graphviz.unflatten).

**Comment by GordonSmith** [comment]
Quick answer is no.

If this is something you think should be added, please open a new issue requesting it.  Some sample DOT before and after (and the CLI arguments used) would help greatly.

**Reply by borango** [reply]
I am also interested in an Ecmascript (wasm?) port of [Graphviz's unflatten](https://graphviz.org/docs/cli/unflatten/) tool.

If this is not ranked high in @GordonSmith 's backlog then I will probably write something equivalent in Ecmascript. After all, the [original source code](https://gitlab.com/graphviz/graphviz/-/blob/main/cmd/tools/unflatten.c) is not very complex.

**Reply by GordonSmith** [reply]
@borango I don't believe an issue / request was ever opened...

**Reply by borango** [reply]
OK sure... I created one now: https://github.com/hpcc-systems/hpcc-js-wasm/issues/149
Thank you Gordon.
